query/query.go

```diff
@@ -110,6 +113,56 @@ func (r *GormQueryRepository) Create(ctx context.Context, q *Query) error {
 	return nil
 }
 
+// Save update the given query
+func (r *GormQueryRepository) Save(ctx context.Context, q Query) (*Query, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "query", "save"}, time.Now())
+	if strings.TrimSpace(q.Title) == "" {
+		return nil, errors.NewBadParameterError("query title cannot be empty string", q.Title).Expected("non empty string")
+	}
+	lbl := Query{}
```

Comment:
please change `lbl` to a meaningful variable

Reply:
Done. 5428a276bd89344b5f93ad49c70e8db523c02958